### DISCUSSION TITLE
ow to install plugin on clappr

### DISCUSSION THREAD

**Discussion by Mnshawaty** [kslimani/clappr-ima-plugin#52] [Q&A]
hi i am using PHP server 
and i am running a live stream on it

how i can use your plugin in my project
i dont know any thing in node and python

help me please

**Comment by kslimani** [comment]
Take a look at the source code on [demo page](https://kslimani.github.io/clappr-ima-plugin/).

Also read [the documentation](https://github.com/kslimani/clappr-ima-plugin#getting-started).
